lib/phoenix_live_reload/channel.ex

```diff
@@ -87,4 +107,26 @@ defmodule Phoenix.LiveReloader.Channel do
 
   defp remove_leading_dot("." <> rest), do: rest
   defp remove_leading_dot(rest), do: rest
+
+  defp web_console_logger_enabled?(socket) do
+    socket.endpoint.config(:live_reload)[:web_console_logger] == true
+  end
+
+  defp join_info(socket) do
+    if url = editor_url(socket) do
+      %{editor_url: url, relative_path: File.cwd!()}
```

Comment:
I am not sure if this will be right in case of umbrellas. I believe in umbrella we will point to the umbrella root but the heex annotations will be specific to the app?